test/controllers/api/v2/host_subscriptions_controller_test.rb

```diff
@@ -75,6 +75,13 @@ def test_auto_attach
       assert_template 'api/v2/host_subscriptions/index'
     end
 
+    def test_auto_attach_simple_content_access
+      @host.organization.stubs(:simple_content_access?).returns(true)
+      if @host.organization.simple_content_access?
```

Comment:
Ditto on the if statement

Reply:
This test also appears to not test anything - it does not issue a request for auto attach. Did you try running it locally with ktest?

Reply:
Yes. I did run it locally with ktest and it passed. Let me know if I am understanding this right. I should be issuing a request for auto attach instead of checking this `@host.organization.simple_content_access?` and asserting for a bad request.